src/models/mobileservices/mobileservice.js

```diff
@@ -140,8 +140,20 @@ export class MobileService {
     return configExtItems;
   }
 
+  /**
+   * Return the docuemntation URL for this service. Firs try to load from an enviroment variable,
+   * otherwise default to the upstream URL.
+   *
+   * @returns the service documentation URL
+   * @memberof MobileService
+   */
   getDocumentationUrl() {
-    return this.customResourceClass.getDocumentationUrl();
+    let url = this.data.documentationUrl;
+    if (this.data.documentationUrl && !this.data.documentationUrl.startsWith('http')) {
```

Comment:
would 
```
 if (this.data.documentationUrl && !this.data.documentationUrl.startsWith('http')) {
      url = `https://${url}`;
    }

    return url || this.customResourceClass.getDocumentationUrl();
```
be more readable if it was written as
```
 if (this.data.documentationUrl && !this.data.documentationUrl.startsWith('http')) {
      url = `https://${url}`;
    } else {
      url = this.customResourceClass.getDocumentationUrl();
    }
    return url;
```

Reply:
It's a little bit more readable but it doesn't cover all cases.

If `this.data.documentationUrl` is "https://www.foobar.com", the `else` case would be entered and `this.customResourceClass.getDocumentationUrl();` would be returned even though the URL was perfectly valid.